### PR TITLE
Added steps to add/remove custom device

### PR DESCRIPTION
**Pull request by pankajparashar**
Rendered article section for review:
Public preview: 
https://github.com/pankajparashar/edge-developer/blob/patch-5/microsoft-edge/devtools-guide-chromium/device-mode/index.md#add-a-custom-mobile-device
Internal fully rendered:
https://review.learn.microsoft.com/microsoft-edge/devtools-guide-chromium/device-mode/?branch=pr-en-us-2925#add-a-custom-mobile-device

AB#47723374

### CODE REVIEW

File: microsoft-edge/devtools-guide-chromium/device-mode/index.md [hunk 1]
```diff
@@ -176,6 +178,11 @@ If the mobile device option that you need isn't included on the default list, yo
 
    ![Adding a custom device](./index-images/device-mode-toggle-device-toolbar-settings-emulated-devices-add.png)
 
+1. Click **Add** to add the new custom device. The new device should now be available in the **Device** list.
```
Comment (mikehoffms) [inline microsoft-edge/devtools-guide-chromium/device-mode/index.md:181]:
```suggestion
1. Click **Add** to add the new custom device. The new device is now available in the **Device** list.
```
simplified wording, optimistic (happy path; assume success) wording
https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/s/should-vs-must

File: microsoft-edge/devtools-guide-chromium/device-mode/index.md [hunk 2]
```diff
@@ -176,6 +178,11 @@ If the mobile device option that you need isn't included on the default list, yo
...
+
+To remove a custom device:
+
+1. Under **Emulated Devices**, in the **Devices** list, hover over the custom device name and click the trash icon to delete the custom device.
```
Comment (mikehoffms) [inline microsoft-edge/devtools-guide-chromium/device-mode/index.md:185]:
```suggestion
1. Under **Emulated Devices**, in the **Devices** list, hover over the custom device name, and then click the trash icon.
```
neither term is in style guide
https://learn.microsoft.com/en-us/search/?scope=Style%20guide&terms=trash
https://learn.microsoft.com/en-us/search/?scope=Style%20guide&terms=wastebasket

https://ahdictionary.com/word/search.html?q=trashcan

File: microsoft-edge/devtools-guide-chromium/device-mode/index.md (hunk 1 above)
Comment (mikehoffms) [reply microsoft-edge/devtools-guide-chromium/device-mode/index.md:181]: applied locally in VS Code

**Review by mikehoffms — APPROVED** [review]
Writer/Editor pass; pushed a commit.

**Review by captainbrosset — APPROVED** [review]

**Pull request merged**